### DISCUSSION TITLE
What do {{date}} {{time}} and {{title}} in your templates?

### DISCUSSION THREAD

**Discussion by liutiming** [BryanJenksCommunity/FAQ#13] [Q&A] [answered]
### Question

What is your question?

What is the function of curly brackets in your templates and what does run `,date` do?
---

By filing a question to this repo, I promise that

-   [x] I have searched open/closed issues on this repo for my answer already

I understand that my issue may be closed if I don't fulfill my promises.

**Comment by tallguyjenks** [comment, marked as answer]
those are variables for the template plugin

as of now there are only 3 if i remember correctly

`{{date}}`, `{{time}}`, `{{title}}`

in the settings pane you can set the format for the date/time variables so i keep mine set to ISO standard:

date: `YYYY-MM-DD`
time: `HH:mm`
